src/lib/viewers/controls/color-picker/ColorPickerPalette.scss

```diff
@@ -0,0 +1,43 @@
+@import '~box-ui-elements/es/styles/variables';
+@import './ColorPickerControl';
+@import '../../../Controls';
+
+.bp-ColorPickerPalette {
+    position: absolute;
+    top: calc(-1 * (#{$controls-bar-bottom-offset} + #{$color-picker-control-height / 2} + 10px)); // 10px is the padding between the popup and the rest of the control bar
+    right: 50%;
+    z-index: 1;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding: 0 8px;
+    background: $controls-background;
```

Comment:
SCSS variables are global and should be namedspaced to avoid collisions in case they're imported for some reason. Let's use `$bp-controls-background`?